aiohttp/http_websocket.py

```diff
@@ -652,8 +654,7 @@ async def _send_frame(
         else:
             header = PACK_LEN3(0x80 | rsv | opcode, 127 | mask_bit, msg_length)
         if use_mask:
-            mask_int = self.randrange(0, 0xFFFFFFFF)
-            mask = mask_int.to_bytes(4, "big")
+            mask = PACK_RANDBITS(self.get_random_bits())
```

Comment:
pack is faster than `to_bytes`. Its already used above in a lot of places but this one wasn't for some reason.